python_env.yml

```diff
@@ -0,0 +1,25 @@
+name: python_nuclear_speckle_env
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.9.12
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::pip
+  - conda-forge::opencv
+  - conda-forge::tifffile
+  - conda-forge::jupyterlab
+  - conda-forge::pandas=1.4.4
+  - conda-forge::ipykernel
+  - conda-forge::nb_conda_kernels
+  - conda-forge::scipy=1.10.0
+  - conda-forge::numpy=1.22
```

Comment:
Consider using later versions of numpy if possible (`>=1.26.x`).

Reply:
I think this had to do with seaborn not being compatible with newer versions of numpy. Will need to look into.